cfme/tests/networks/test_sdn_balancers.py

```diff
@@ -68,6 +68,7 @@ def test_sdn_balancers_detail(provider, network_prov_with_load_balancers):
 
 
 # only one provider is needed for that test, used Azure as it has balancers
+@pytest.mark.ignore_stream("5.11", "upstream")  # BZ 1672949 (RFE)
```

Comment:
Is that only test you need to ignore or all the `test_sdn_balancers`?

Reply:
@kedark3 good catch! it's the whole module that needs to be blocked

Reply:
updated

Reply:
Awesome I approve.